src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -116,6 +122,28 @@
               optional = true)
     private Boolean overwriteExistingWorkspace = false;
 
+    @Argument(fullName = BATCHSIZE_ARG_NAME,
+              shortName = BATCHSIZE_ARG_NAME,
+              doc = "Batch size controls the number of samples for which readers are open at once " +
+                    "and therefore provides a way to minimize memory consumption. However, it can take longer to complete. " +
+                    "Use consolidate flag if more than hundred batches were used. This will improve feature read time. " +
+                    "batchSize=0 means no batching. Readers for all samples will be opened at once. " +
```

Comment:
"batchSize=0 means no batching. Readers for all samples will be opened at once. " -> "batchSize=0 means no batching (ie., readers for all samples will be opened at once)"